exercises/transpose/transpose_test.py

```diff
@@ -0,0 +1,275 @@
+import unittest
+from transpose import transpose
+
+
```

Comment:
Could you add the test data version (`# test cases adapted from `x-common//canonical-data.json` @ version: 1.0.0`) here, like here: https://github.com/exercism/xpython/blob/master/exercises/hello-world/hello_world_test.py#L6